packages/notebook/src/panel.ts

```diff
@@ -84,7 +84,10 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     });
   }
 
-  _onSave(sender: DocumentRegistry.Context, state: DocumentRegistry.SaveState) {
+  _onSave(
```

Comment:
We should also make that method `private` for 4.0, and can be tracked in a different issue.

Reply:
Opened https://github.com/jupyterlab/jupyterlab/pull/10420